locale/translations/it_IT.json

```diff
@@ -113,6 +113,7 @@
     "page.about.build_date": "Data della build:",
     "page.about.author": "Autore:",
     "page.about.license": "Licenza:",
+    "page.about.global_config_options": "opzioni di configurazione globali",
```

Comment:
Same thing as other similar languages. First letter should start with an uppercase.